.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+Fixes/delivers issue #<issue_number>
```

Comment:
this doesn't follow the [closing keywords syntax ](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

I would prefer to make example which fits:
`fixes dotnet/templating#<issue number>`